server/lib/logger.ts

```diff
@@ -125,6 +150,35 @@ const checkFiles = () => {
   }
 };
 
+/**
+ * Checks if the wazuh-frontend.log file size is greater than 100MB, if so it rotates the file.
+ */
+ const checkFrontendLogFiles = () => {
+  if (allowed) {
+    if (getFilesizeInMegaBytes(WAZUH_FRONTEND_LOGS_RAW_PATH) >= 100) {
+      fs.renameSync(
+        WAZUH_FRONTEND_LOGS_RAW_PATH,
+        `${WAZUH_DATA_LOGS_DIRECTORY_PATH}/wazuh-frontend.${new Date().getTime()}.log`
+      );
+      fs.writeFileSync(
+        WAZUH_FRONTEND_LOGS_RAW_PATH,
+        JSON.stringify({
+          date: new Date(),
+          level: 'info',
+          location: 'logger',
+          message: 'Rotated log file'
+        }) + '\n'
+      );
+    }
+    if (getFilesizeInMegaBytes(WAZUH_FRONTEND_LOGS_PLAIN_PATH) >= 100) {
+      fs.renameSync(
+        WAZUH_FRONTEND_LOGS_PLAIN_PATH,
+        `${WAZUH_DATA_LOGS_DIRECTORY_PATH}/wazuh-frontend-plain.${new Date().getTime()}.log`
+      );
+    }
+  }
+};
+
```

Comment:
move all frontend logs related to a new file to avoid superclasses

Reply:
created new file `ui-logger`